dandi/tests/test_models.py

```diff
@@ -116,9 +116,9 @@ def test_asset():
         (
             LicenseType,
             {
-                "CC0": "dandi:CC0",
-                "CCBY40": "dandi:CCBY40",
-                "CCBYNC40": "dandi:CCBYNC40",
+                "CC0_10": "dandi:CC0-1.0",
+                "CC_BY_40": "dandi:CC-BY-4.0",
+                "CC_BY_NC_40": "dandi:CC-BY-NC-4.0",
```

Comment:
BTW! should we may be define `spdx` prefix and point to https://spdx.org/licenses/ for this purpose?  just tested on a random one: https://spdx.org/licenses/CC-BY-3.0 works

Reply:
we could use spdx directly. i'll make the change